src/Nethermind/Nethermind.JsonRpc/Modules/Eth/SimulateTxExecutor.cs

```diff
@@ -249,10 +255,34 @@ private int MapSimulateErrorCode(TransactionResult txResult)
             };
         }
 
-        return txResult.EvmExceptionType switch
-        {
-            EvmExceptionType.Revert => ErrorCodes.RevertedSimulate,
-            _ => ErrorCodes.VMError
-        };
+        return MapEvmExceptionType(txResult.EvmExceptionType);
     }
+
+
+    private static int MapEvmExceptionType(EvmExceptionType type) => type switch
+    {
+        EvmExceptionType.Revert => ErrorCodes.RevertedSimulate,
+        _ => ErrorCodes.VMError
+    };
+
+    private static string MapEvmExceptionTypeToMessage(EvmExceptionType type) => type switch
```

Comment:
Should it be different from Nethermind.Evm.EvmExceptionExtensions.GetEvmExceptionDescription mapping?